DOCS-15667 Updates to view & analyze docs page (#97)
mongodb/docs-landing: source/view-analyze.txt

   .. card::
      :headline: Atlas
      :url: https://www.mongodb.com/docs/atlas/
      :icon: /images/icons/atlas.svg
      :icon-alt: Atlas LogoMark

      Discover how to query and update data from within Atlas, MongoDB's fully
      managed cloud database.

   .. card::
      :headline: Atlas Data Federation
      :url: https://www.mongodb.com/docs/atlas/data-federation/
      :icon: /images/icons/data-federation.svg
      :icon-alt: Atlas Data Federation LogoMark

      Query and analyze data across Atlas Clusters, Amazon S3 Buckets,
      and Atlas Data Lake Datasets with the MongoDB Query Language. 

   .. card::
      :headline: Atlas Search
      :url: https://www.mongodb.com/docs/atlas/atlas-search/
      :icon: /images/icons/text-search.svg
      :icon-alt: Atlas Search LogoMark

      Build fast, relevant, full-text search capabilities on top of your data in
      the cloud.

   .. card::
      :headline: Atlas Data Lake
      :url: https://www.mongodb.com/docs/datalake/
      :icon: /images/icons/data-lake.svg
      :icon-alt: Atlas Data Lake LogoMark

>       Utilize Data Lake to create analytic optimized copies of your

Reviewer: ```suggestion
      Use Data Lake to create analytic optimized copies of
```

>       Atlas Cluster data with complete workload isolation.

Reviewer: ```suggestion
      your data with complete workload isolation.
```

   .. card::
      :headline: Atlas SQL
      :url: https://www.mongodb.com/docs/atlas/data-federation/query/query-with-sql/
      :icon: /images/icons/atlas-sql.svg
      :icon-alt: Atlas SQL LogoMark

>       Create and run SQL queries to visualize, graph, and report on your

Reviewer: suggestion: can we trim this somehow? it's length is causing extra whitespace to appear in the card next to it.

>       Atlas data using relational business intelligence tools.

Reviewer: ```suggestion
      data using relational business intelligence tools.
```
